src/commands/tick-log.js

```diff
@@ -40,10 +43,12 @@ function log(argv) {
   let { start, end } = parseDateRange(argv.date)
   start = moment(start).toArray()
   end = moment(end).toArray()
+  const filter = argv.filter ? compileFilter(argv.filter) : null 
+  const opts = filter ? { filter } : { start, end, tags: argv.tag }
```

Comment:
Does it make sense to have this `tag: argv.tag` here since it's been changed to use filter?